release_notes/365-release-notes.adoc

```diff
@@ -0,0 +1,87 @@
+[id="365-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.65
+include::modules/common-attributes.adoc[]
+:context: 365-release-notes
+
+toc::[]
+
+{product-title} (RHACS) 3.65 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* September 6, 2021
+
+[id="new-features"]
+== New features
+
+[id="mitre-attck-framework"]
+=== MITRE ATT&CK framework
+You can now use the link:https://attack.mitre.org/matrices/enterprise/containers/[MITRE ATT&CK Framework] to categorize policies in the {product-title}. When you xref:../operating/manage-security-policies.adoc#create-custom-policies[create custom security policies], you can now add MITRE ATT&CK framework's adversary tactics and techniques related information.
+
+[id=""]
+=== Install on more platforms
+You can now install {product-title} on:
+
+* link:https://aws.amazon.com/rosa/[Red Hat OpenShift Service on AWS (ROSA)]
+* link:https://azure.microsoft.com/en-au/services/openshift/#overview[Azure Red Hat OpenShift]
+
+[id="admission-control-configuration"]
+=== Admission control configuration
+You can now configure the dynamic admission control settings in the {product-title} Operator. It now includes the following new admission control settings:
+
+* `admissionControl.bypass`: Use this parameter to bypass the admission controller.
+* `admissionControl.contactImageScanners`: Specify `true` to enable inline scanning of images not already scanned during a deployment's admission review.
+* `admissionControl.timeoutSeconds`: Use this parameter to specify the maximum number of seconds {product-title} should wait for an admission review before marking it as fail open.
```

Comment:
nit: These three bullets begin with verbs, but can you revise to start them with the same verb?  The first one, `bypass`, doesn't need to be set to `true`?

Friendly reminder that if you accepted the sugg for the desc to `contactImageScanners` in the other file, consider repeating here.  I guess it's a general reminder to make the descs the same in both files.

Reply:
It looked monotonous, I don't know. I can change if you want me to. Your call.